website/components/Head/index.js

```diff
@@ -0,0 +1,21 @@
+import NextHead from 'next/head';
+import PropTypes from 'prop-types';
+
+const Head = ({ meta }) => {
+  return (
+    <NextHead>
+      <title>{meta?.title}</title>
+      <meta content={meta?.description} name="description" />
+      <meta property="og:type" content={meta?.type} />
+      <meta property="og:site_name" content="Strapi Design System" />
+      <meta property="og:description" content={meta?.description} />
+      <meta property="og:title" content={meta?.title} />
+    </NextHead>
+  );
+};
+
+Head.protoTypes = {
```

Comment:
I think it's `propTypes` here instead of **protoTypes**

Can describe the shape of the object?

```js
Head.propTypes = {
   meta: PropTypes.shape({ description: PropTypes.string, title....})
}
```

And also don't forget about the defaultProps:


```js
Head.defaultProps = { }
```